board/stm32fx/clock.h

```diff
@@ -1,4 +1,8 @@
-void clock_init(void) {
+bool failed_clock_init = false;
```

Comment:
should only be defined once. could also use a better name that makes it clear we're in a "limited" mode - current name is a bit misleading when reading code that references it in other places